gnovm/pkg/gnolang/internal/softfloat/gen/main.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+func main() {
+	// Process softfloat64.go file
+	processSoftFloat64File()
+
+	// Process softfloat64_test.go file
+	processSoftFloat64TestFile()
+
+	// Run mvdan.cc/gofumpt
+	gofumpt()
+
+	fmt.Println("Files processed successfully.")
+}
+
+func processSoftFloat64File() {
+	// Read source file
+	content, err := os.ReadFile(fmt.Sprintf("%s/src/runtime/softfloat64.go", runtime.GOROOT()))
+	if err != nil {
+		log.Fatal("Error reading source file:", err)
+	}
+
+	// Prepare header
+	header := `// Code generated by github.com/gnolang/gno/gnovm/pkg/gnolang/internal/softfloat/gen. DO NOT EDIT.
+// This file is copied from $GOROOT/src/runtime/softfloat64.go.
+// It is the software floating point implementation used by the Go runtime.
+
+`
+
+	// Combine header with content
+	newContent := header + string(content)
+
+	// Replace package name
+	newContent = strings.Replace(newContent, "package runtime", "package softfloat", 1)
+
+	// Write to destination file
+	err = os.WriteFile("runtime_softfloat64.go", []byte(newContent), 0o644)
+	if err != nil {
+		log.Fatal("Error writing to destination file:", err)
+	}
+}
+
+func processSoftFloat64TestFile() {
+	// Read source test file
+	content, err := os.ReadFile(fmt.Sprintf("%s/src/runtime/softfloat64_test.go", runtime.GOROOT()))
+	if err != nil {
+		log.Fatal("Error reading source test file:", err)
+	}
+
+	// Prepare header
+	header := `// Code generated by github.com/gnolang/gno/gnovm/pkg/gnolang/internal/softfloat/gen. DO NOT EDIT.
+// This file is copied from $GOROOT/src/runtime/softfloat64_test.go.
+// It is the tests for the software floating point implementation
+// used by the Go runtime.
+
+`
+
+	// Combine header with content
+	newContent := header + string(content)
+
+	// Replace package name and imports
+	newContent = strings.Replace(newContent, "package runtime_test", "package softfloat_test", 1)
+	newContent = strings.Replace(newContent, "\t. \"runtime\"", "\t\"runtime\"", 1)
+	newContent = strings.Replace(newContent, "GOARCH", "runtime.GOARCH", 1)
+
+	newContent = strings.Replace(newContent, "import (", "import (\n\t. \"github.com/gnolang/gno/gnovm/pkg/gnolang/internal/softfloat\"", 1)
+
+	// Write to destination file
+	err = os.WriteFile("runtime_softfloat64_test.go", []byte(newContent), 0o644)
+	if err != nil {
+		log.Fatal("Error writing to destination test file:", err)
+	}
+}
+
+func gofumpt() {
+	rootPath, err := exec.Command("git", "rev-parse", "--show-toplevel").Output()
```

Comment:
https://github.com/gnolang/gno/blob/master/misc/genstd/util.go#L66-L87

This doesn't require callling an `exec` on git